bosshud/ze_amalgamation_v1_1.txt

```diff
@@ -3,21 +3,21 @@
 	"0"
 	{
 		"HP_counter"		"level_1_mb_hp_counter"
-		"CustomText"		"Mini Boss"
+		"CustomText"		"Crystal"
 	}
 	"1"
 	{
 		"HP_counter"		"boss_hp_p1"
-		"CustomText"		"Boss Phase 1"
+		"CustomText"		"Crashdab - Phase 1"
 	}
 	"2"
 	{
 		"HP_counter"		"boss_hp_p2"
-		"CustomText"		"Boss Phase 2"
+		"CustomText"		"Crashdab - Phase 2"
 	}
 	"3"
 	{
 		"HP_counter"		"final_boss_hp_counter"
-		"CustomText"		"Boss"
+		"CustomText"		"BOSS"
```

Comment:
Just remove this line altogether, like I've told you before don't use CustomText unless you're actually adding something custom.